test/contracts/OVM/verification/OVM_FraudVerifier.spec.ts

```diff
@@ -366,6 +378,163 @@ describe('OVM_FraudVerifier', () => {
+        it('creates multiple state transitioners per tx hash', async () => {
+          await expect(
+            OVM_FraudVerifier.initializeFraudVerification(
+              NULL_BYTES32,
+              DUMMY_BATCH_HEADERS[0],
+              DUMMY_BATCH_PROOFS[0],
+              DUMMY_OVM_TRANSACTIONS[1],
+              DUMMY_TX_CHAIN_ELEMENTS[0],
+              DUMMY_BATCH_HEADERS[0],
+              DUMMY_BATCH_PROOFS[0]
+            )
+          ).to.not.be.reverted
+
+          expect(
+            await OVM_FraudVerifier.getStateTransitioner(
+              NULL_BYTES32,
+              DUMMY_HASH
+            )
+          ).to.equal(Mock__OVM_StateTransitioner.address)
+          expect(
+            await OVM_FraudVerifier.getStateTransitioner(
+              NULL_BYTES32,
+              DUMMY_HASH_2
+            )
+          ).to.equal(state2.address)
+        })
+
+        const batchProof = {
+          ...DUMMY_BATCH_PROOFS[0],
+          index: DUMMY_BATCH_PROOFS[0].index + 1,
+        }
+
+        // 1. after a successful fraud proof from pre-execution state s, the transaction is removed and a new (potentially unrelated) transaction will now have pre-execution state s.
+        it('Case 1: allows proving fraud on the same pre-state root twice', async () => {
+          // finalize previous fraud
+          await OVM_FraudVerifier.finalizeFraudVerification(
+            NULL_BYTES32,
+            DUMMY_BATCH_HEADERS[0],
+            DUMMY_BATCH_PROOFS[0],
+            DUMMY_HASH,
+            NON_NULL_BYTES32,
+            DUMMY_BATCH_HEADERS[0],
+            batchProof
+          )
+
+          // start new fraud
+          await OVM_FraudVerifier.initializeFraudVerification(
+            NULL_BYTES32,
+            DUMMY_BATCH_HEADERS[0],
+            DUMMY_BATCH_PROOFS[0],
+            DUMMY_OVM_TRANSACTIONS[1],
+            DUMMY_TX_CHAIN_ELEMENTS[1],
+            DUMMY_BATCH_HEADERS[1],
+            DUMMY_BATCH_PROOFS[1]
+          )
+
+          // finalize it as well
+          await OVM_FraudVerifier.finalizeFraudVerification(
+            NULL_BYTES32,
+            DUMMY_BATCH_HEADERS[0],
+            DUMMY_BATCH_PROOFS[0],
+            DUMMY_HASH_2,
+            NON_NULL_BYTES32,
+            DUMMY_BATCH_HEADERS[1],
+            batchProof
+          )
+
+          // the new batch was deleted
+          expect(
+            Mock__OVM_StateCommitmentChain.smocked.deleteStateBatch.calls[0]
+          ).to.deep.equal([
+            Object.values(DUMMY_BATCH_HEADERS[1]).map((value) => {
+              return Number.isInteger(value) ? BigNumber.from(value) : value
+            }),
+          ])
+        })
+
+        // 2. after successful fraud proof that deletes a state with an ongoing (or completed) fraud verification, the new Canonical Transaction chain could be constructed to recreate the same state.
+        it.skip('Case 2: concurrent fraud proofs', async () => {
+          // ?
+        })
```

Comment:
I looked over what I had originally intended and there's no way it could actually do what I thought, we can remove this!